charts/registry-creds/templates/secrets_acr.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if and .Values.acr.enabled (not .Values.acr.existingSecretName) }}
```

Comment:
`acr.existingSecretName` should be defined in `values.yaml` and `README.md`

Reply:
Thanks. I just push new commit to fix them.

Reply:
Thanks! You are still missing the updates to https://github.com/kir4h/charts/blob/master/charts/registry-creds/templates/deployment.yaml (saying just in case you didn't see my other comment in the general conversation).

Last thing: could you please open the PR against the `devel` branch instead of `master`? I just created it because I noticed CI is broken so I need to do an extra commit there before releasing.

Reply:
Thanks for the feedback. Sure, I will fix them and open another PR to `devel` branch.

Reply:
> Thanks! You are still missing the updates to https://github.com/kir4h/charts/blob/master/charts/registry-creds/templates/deployment.yaml (saying just in case you didn't see my other comment in the general conversation).
> 
> Last thing: could you please open the PR against the `devel` branch instead of `master`? I just created it because I noticed CI is broken so I need to do an extra commit there before releasing.

Close this MR based on discussion above.